### DISCUSSION TITLE
problema  de instalaçao do meu yarn no uikit

### DISCUSSION THREAD

**Discussion by lucccansousa** [balta-io/forum#339] [Tira Dúvidas] [answered]
![image](https://github.com/balta-io/forum/assets/131711446/70106e0d-153d-4a61-aa23-2ba474382c66)

quero ajuda nisso pfv

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @lucccansousa, como vai?

Qual curso/aula está fazendo? Quais procedimentos está executando até chegar neste erro?
Por favor, detalha mais para que possamos te ajudar.

**Reply by lucccansousa** [reply]
to fazendo de angular front-end

**Reply by lucccansousa** [reply]
https://yarnpkg.com/en/docs/cli/run

n consigo e instalar o yarn no meu projeto

**Reply by andrebaltieri** [reply]
Bom dia, @lucccansousa, como vai?

Você está utilizando as mesmas versões do curso? Isto pode mudar com o tempo.
Em adicional, o que apareceu de erro na sua imagem foi que o PNPM não pode ser encontrado.

Talvez instalar ele antes de rodar os comandos resolva.
https://pnpm.io/cli/install
